src/main/scala/ScalaLanguageClient.scala

```diff
@@ -96,4 +96,49 @@ class ScalaLanguageClient extends AutoLanguageClient { client =>
       matching.foreach { _.dismiss() }
     }
   }
+  
+  override def postInitialization(server: ActiveServer): Unit = {
+    logger.debug(s"Server Capabilities:\n${js.JSON.stringify(server.capabilities, space = 2)}")
```

Comment:
Remove debug logging before merging, please. Btw, you can see it in the Atom logs anyway.